v2/elasticsearch-common/src/main/java/com/google/cloud/teleport/v2/elasticsearch/options/package-info.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (C) 21 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+/**
+ * Package info for elasticsearch options.
+ */
+package com.google.cloud.teleport.v2.elasticsearch.options;
```

Comment:
Please move to `com.google.cloud.teleport.v2.options` package (no "elasticsearch" in the package name).